tests/modules/GoldenEggMetadata.t.sol

```diff
@@ -69,8 +70,13 @@ contract GoldenEggMetadataTests is TestConfig {
     }
 
     function test_getGoldenEggTokenId(uint32 editionCutoffTime, uint32 mintQuantity) external {
-        vm.assume(mintQuantity > 0 && mintQuantity < 10);
-        vm.assume(editionCutoffTime > block.timestamp);
+        if (!(mintQuantity > 0 && mintQuantity < 10)) {
+            mintQuantity = 5;
+        }
```

Comment:
I'm curious - why the need for this change?

Reply:
For better fuzz coverage.